exaudfclient/base/script_options_parser/ctpg/script_option_lines_ctpg.cc

```diff
@@ -238,9 +236,18 @@ void parseOptions(const std::string& code, options_map_t & result) {
     std::optional<LinePositions> currentLinePositions = getNextLine(current_pos, code);
     while (currentLinePositions) {
 
-        std::string line = code.substr(currentLinePositions->mStartPos, currentLinePositions->mEndPos);
+        std::string line = code.substr(currentLinePositions->mStartPos,
+                                        currentLinePositions->mEndPos - currentLinePositions->mStartPos);
```

Comment:
This was actually a bug.
It's std::string::subst(begin, length), not std::string::subst(begin, end)

Reply:
does this change something on the performance?

Reply:
Here results of our performance tests:
Master branch: 6.9s
This branch: 7.0s